backend/server/models.py

```diff
@@ -103,18 +103,38 @@ def load_estimator(self) -> BaseMLEstimator:
 
 
 class Prediction(models.Model):
+    """Model for ML model predictions for each match"""
+
     match = models.ForeignKey(Match, on_delete=models.CASCADE)
     ml_model = models.ForeignKey(MLModel, on_delete=models.CASCADE)
-    predicted_winner = models.ForeignKey(Team, on_delete=models.CASCADE)
+    predicted_winner = models.ForeignKey(
+        Team, on_delete=models.CASCADE, related_name="predicted_wins"
+    )
     predicted_margin = models.PositiveSmallIntegerField()
+    is_correct = models.BooleanField(default=False)
+
+    @classmethod
+    def calculate_is_correct(cls, match: Match, predicted_winner: Team) -> bool:
```

Comment:
I dont know about your name conventions, but if this function returns a bool, could be `is_calculate_correct` instead of `calculare_is_correct` more indicative of boolean?

Reply:
It doesn't represent the field/attribute itself, but the calculation of the correct value that then gets assigned to a given record. It might be better as an instance method, but I often want to calculate the value during the instance creation step, so I use the class object to do that.

I think a better name might be `calculate_whether_correct`